### PR TITLE
non editable configs for permissionless markets

### PR DESCRIPTION
**Pull request by nope-finance**

### CODE REVIEW

File: token-lending/program/src/processor.rs [hunk 1]
```diff
@@ -2010,6 +2015,29 @@ fn process_update_reserve_config(
...
+            return Err(LendingError::InvalidConfig.into());
+        }
+        if reserve.config.fees.flash_loan_fee_wad != config.fees.flash_loan_fee_wad {
+            msg!("permissionless markets can't edit flash loan fees");
+            return Err(LendingError::InvalidConfig.into());
+        }
+        if reserve.config.fees.host_fee_percentage != config.fees.host_fee_percentage {
+            msg!("permissionless markets can't edit host fee percentage");
+            return Err(LendingError::InvalidConfig.into());
+        }
```
Comment (0xripleys) [inline token-lending/program/src/processor.rs:2087]: is it intentional that origination fees can be changed by permissionless pool owner?

Comment (nope-finance) [reply token-lending/program/src/processor.rs:2087]: yeh

Comment (0xripleys) [reply token-lending/program/src/processor.rs:2087]: tests are failing bc i think we try to set one of these config params with a non-5ph key. apart from that, lgtm

Comment (nope-finance) [reply token-lending/program/src/processor.rs:2087]: yeh i was gonna fix that soon

**Review by 0xripleys — APPROVED** [review]

**Pull request merged**